src/Sentry.Unity/TimeDebounceBase.cs

```diff
@@ -45,4 +45,12 @@ internal sealed class ErrorTimeDebounce : TimeDebounceBase
     {
         public ErrorTimeDebounce(TimeSpan debounceOffset) => _debounceOffset = debounceOffset;
     }
+
+    /// <summary>
+    /// This class is not thread-safe and is designed to be called by Unity non-threaded logger callback
+    /// </summary>
+    internal sealed class WarningTimeDebounce : TimeDebounceBase

```

Comment:
Can we get this in through a PR first?
Lets drop the release part of this PR and do it through the process we agreed on #75